cumulus_logger.py

```diff
@@ -39,19 +39,24 @@ def __getExceptionMessage(self, **kwargs):
             exceptionInfo = kwargs['exc_info'] if isinstance(kwargs['exc_info'], tuple) else sys.exc_info()
             exceptionStr = ' ' + logging.Formatter().formatException(exceptionInfo)
         return exceptionStr
+    
+    def __getEvent(self, event):
```

Comment:
This seems like a bad name for a function that fetches the execution name.

Reply:
80b26bb